lib/message/message.js

```diff
@@ -134,15 +134,15 @@ Message.fromJSON = function fromJSON (json) {
   if (JSUtil.isValidJSON(json)) {
     json = JSON.parse(json)
   }
-  return new Message(json.message)
+  return new Message(json.messageBuffer)
 }
 
 /**
  * @returns {Object} A plain object with the message information
  */
 Message.prototype.toObject = function toObject () {
   return {
-    message: this.messageBuffer.toString()
+    messageBuffer: this.messageBuffer.toString('base64')
```

Comment:
@ryanxcharles Is this what you had in mind ultimately? Your comment indicated to just drop the `toString`, but then that led to an error.

Reply:
this isn't quite what i had in mind, but this is fine. probably the "toObject" really just doesn't even make sense anyway. but this solves the issue.

Reply:
this is not right ... i think it should just be a buffer. your tests mix and match base64 strings.